exonum/src/lib.rs

```diff
@@ -23,6 +23,11 @@
 #![cfg_attr(feature = "long_benchmarks", feature(test))]
 #![cfg_attr(feature = "cargo-clippy", deny(clippy_pedantic))]
 #![cfg_attr(feature = "cargo-clippy", allow(stutter, similar_names, items_after_statements))]
+#![cfg_attr(feature = "cargo-clippy", allow(use_self, cast_possible_truncation,
+                                            replace_consts, enum_glob_use, cast_sign_loss,
+                                            used_underscore_binding, cast_possible_wrap,
+                                            single_match_else, option_map_unwrap_or_else,
+                                            option_map_unwrap_or, filter_map))]
 
```

Comment:
The list of the lints:

use_self
cast_possible_truncation
replace_consts
enum_glob_use
cast_sign_loss
used_underscore_binding
cast_possible_wrap
single_match_else
option_map_unwrap_or_else
option_map_unwrap_or
filter_map